feature/policy_forwarding/GUE/Outer_IPv4/decapsulation/otg_tests/Inner_IP/README.md

```diff
@@ -0,0 +1,337 @@
+
+ - Decapsulate the outer (transport) layer 3 and GUE headers of GUE packets destined to locally-configured decap VIP and matching UDP port after that it will forward them based on the exposed inner (payload) L3 header.
+ - GUE Inner protocol type must be derived from a unique DST port. If not specifically configured, then the following default DST UDP port will be used.
+    - For inner IPv4 - GUE UDP port 6080
+    - For inner IPv6 - GUE UDP port 6615
+ - Post decapsulation the DUT should copy outer TTL(and decrement) to inner header and maintain the inner DSCP vaule.
+    - If explicit configration is present to not copy the TTL, then it will be honored. 
+ - Decapsulate the packet only if it matches the locally configured GUE VIP and associated UDP port/port-range.
+    - Traffic not subject to match criteria will be forwared using traditional IP forwarding. 
+ - UDP checksum in the transport (outer) header is used for the packet integrity validation, and the DUT will ignore the GUE checksum (non-zero or all-zero). 
+
+## Testbed type
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+* Create the following connections:
+* DUT has ingress and egress port connected to the ATE.
+  
+```mermaid
+graph LR; 
+A[ATE:Port1] --Ingress--> B[Port1:DUT:Port2];B --Egress--> C[Port2:ATE];
+```
+
+*  ATE Port 1: Generates GUE-encapsulated traffic with various inner (original) destinations.
+*  ATE Port 2: Receives decapsulated traffic whose inner destination matches the policy.
+  
+### DUT Configuration
+
+1.  Interfaces: Configure all DUT ports as singleton IP interfaces.
+ 
+2.  Static Routes/LSPs:
+    *  Configure an IPv4 static route to GUE decapsulation destination (DECAP-DST) to Null0.
+    *  Have policy configuration that match GUE decapsulation destination and default/non-default GUE UDP port/port-range for the decapsulation.
+       *  If udp port is not configured then the default GUE UDP port will be inherited/used.
+    *  Apply the defined policy on the Ingress (DUT port1) port.
+    *  Configure static routes for encapsulated traffic destinations IPV4-DST1 and IPV6-DST1 towards ATE Port 2.
+    *  Configure static MPLS label binding (LBL1) towards ATE Port 2. Next hop of ATE Port 1 should be indicated for MPLS pop action.
+    *  Configure static routes for destination IPV4-DST2 and IPV6-DST2 towards ATE Port 2.
+
+3.  Policy-Based Forwarding: 
+    *  Rule 1: Match GUE traffic with destination DECAP-DST using destination-address-prefix-set and default/non-default GUE UDP port/port-range for decapsulation.
+      * If udp port is not configured then the default GUE UDP port will be inherited/used.   
```

Comment:
Same suggestion as above to add clarifications along what exactly do we want the NOS to do here.

Reply:
do you mean add sample vendor config?